types/react-html-email/index.d.ts

```diff
@@ -0,0 +1,78 @@
+// Type definitions for react-html-email 3.0
+// Project: https://github.com/chromakode/react-html-email#readme
+// Definitions by: Saiichi Hashimoto <https://github.com/saiichihashimoto>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.8
+
+import type { CSSProperties, FunctionComponent, FunctionComponentElement, ReactElement, ReactNode } from 'react';
+
+export const A: FunctionComponent<{
+    children?: ReactNode;
+    color?: string;
+    download?: string;
+    href: string;
+    style?: CSSProperties;
+    textDecoration?: string;
+}>;
+
+export const Box: FunctionComponent<{
+    align?: 'left' | 'center' | 'right';
+    bgcolor?: string;
+    border?: string;
+    cellPadding?: number;
+    cellSpacing?: number;
+    children?: ReactNode;
+    height?: string;
+    style?: CSSProperties;
+    valign?: 'top' | 'middle' | 'bottom';
+    width?: string;
+}>;
+
+export const Email: FunctionComponent<{
+    align?: 'left' | 'center' | 'right';
+    bgcolor?: string;
+    bodyStyle?: CSSProperties;
+    cellPadding?: number;
+    cellSpacing?: number;
+    children?: ReactNode;
+    headCSS?: string;
+    lang?: string;
+    style?: CSSProperties;
+    title: string;
+    valign?: 'top' | 'middle' | 'bottom';
+    width?: string;
+}>;
```

Comment:
Suggestion: it’s often handy for React users to be able to easily reference the props type of components they use. I.e., it would be a much nicer experience if they could `import { EmailProps } from "react-html-email"` instead of having to do `Parameters<typeof Email>[0]`. And it would save you from having to do the same thing at the bottom, and then you’d also be able to drop the minimum TypeScript version required for these types down to the minimum supported version.